Formula/widelands.rb

```diff
@@ -35,23 +36,23 @@ class Widelands < Formula
 
   uses_from_macos "python" => :build
   uses_from_macos "curl"
-
-  # Fix build with Boost 1.77+.
-  # Remove with the next release (1.1).
-  patch do
-    url "https://github.com/widelands/widelands/commit/316eaea209754368a57f445ea4dd016ecf8eded6.patch?full_index=1"
-    sha256 "358cae53bbc854e7e9248bdea0ca5af8bce51e188626a7f366bc6a87abd33dc9"
-  end
+  uses_from_macos "zlib"
 
   def install
     ENV.cxx11
-    system "cmake", "-S", ".", "-B", "build",
-                    # Without the following option, Cmake intend to use the library of MONO framework.
-                    "-DPNG_PNG_INCLUDE_DIR:PATH=#{Formula["libpng"].opt_include}",
-                    "-DWL_INSTALL_DATADIR=#{pkgshare}/data",
-                    # older versions of macOS may not have `python3`
-                    "-DPYTHON_EXECUTABLE=#{which("python3") || which("python")}",
-                    *std_cmake_args
+
+    cmake_args = std_cmake_args + %w[
+      -DOPTION_BUILD_WEBSITE_TOOLS=OFF
+      -DOPTION_BUILD_TESTS=OFF
+      -DCMAKE_INSTALL_PREFIX=#{prefix}
+      -DWL_INSTALL_BASEDIR=#{pkgshare}
+      -DWL_INSTALL_BINDIR=#{bin}
+      -DWL_INSTALL_DATADIR=#{pkgshare}/data
+      -DPNG_PNG_INCLUDE_DIR:PATH=#{Formula["libpng"].opt_include}
+      -DPYTHON_EXECUTABLE=#{which("python3") || which("python")}
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *cmake_args
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *cmake_args, *std_cmake_args
```
Always pass `std_cmake_args` last to make sure you don't override it